script/instance_from_file.py

```diff
@@ -30,6 +30,8 @@ def load_mgr_from_file(mgr: openzwavemqtt.OZWManager, file_path):
 
 
 def camelcase_to_snake_case(name):
+    # Otherwise ZWave -> _z_wave_ in names.
+    name = name.replace("ZWave", "Zwave")
```

Comment:
missing the hyphen in the replace?

Reply:
No, this is on purpose. The property `isZWavePlus` used to be renamed to `is_z_wave_plus` but with this change is renamed to `is_zwave_plus`, which is what we have in the code base too and makes more sense.